### DISCUSSION TITLE
Mac version, no package 'opencv' found

### DISCUSSION THREAD

**Discussion by agi-codes** [lkwq007/stablediffusion-infinity#55] [Q&A]
the error I'm getting when running code on MacOS12.6: 
Compiling and loading c extensions from "/Users/x/stablediffusion-infinity/PyPatchMatch".
rm -rf build/obj libpatchmatch.so
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
No package 'opencv' found
build/obj
build/obj/csrc
[dep] csrc//masked_image.cpp ...
In file included from csrc//masked_image.cpp:1:
csrc/masked_image.h:3:10: fatal error: 'opencv2/core.hpp' file not found
#include <opencv2/core.hpp>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
No package 'opencv' found
[dep] csrc//inpaint.cpp ...
csrc//inpaint.cpp:3:10: fatal error: 'opencv2/imgcodecs.hpp' file not found
#include <opencv2/imgcodecs.hpp>
         ^~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
No package 'opencv' found
[dep] csrc//nnf.cpp ...
In file included from csrc//nnf.cpp:5:
csrc/masked_image.h:3:10: fatal error: 'opencv2/core.hpp' file not found
#include <opencv2/core.hpp>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
No package 'opencv' found
[dep] csrc//pyinterface.cpp ...
In file included from csrc//pyinterface.cpp:1:
csrc/pyinterface.h:1:10: fatal error: 'opencv2/core.hpp' file not found
#include <opencv2/core.hpp>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
Package opencv was not found in the pkg-config search path.
Perhaps you should add the directory containing `opencv.pc'
to the PKG_CONFIG_PATH environment variable
No package 'opencv' found
[CC] csrc//pyinterface.cpp ...
In file included from csrc//pyinterface.cpp:1:
csrc/pyinterface.h:1:10: fatal error: 'opencv2/core.hpp' file not found
#include <opencv2/core.hpp>
         ^~~~~~~~~~~~~~~~~~
1 error generated.
make: *** [build/obj/csrc//pyinterface.o] Error 1
patch_match compiling failed

**Comment by lkwq007** [comment]
You can install `opencv` via homebrew
By the way, the program also works without `opencv`.
